app/services/query-service.js

```diff
@@ -155,16 +162,37 @@ const fetchAnswer = async (req, query, chatHistory, cacheEnabled, summariesEnabl
       onFailedAttempt
     })
 
+  let items = []
+
   if (summariesEnabled) {
     const { response: summariesResponse, hallucinated } = await runFetchAnswerQuery({ query, chatHistory, summariesMode: true, model, embeddings })
     const isResponseValid = validateResponseSummaries(summariesResponse)
 
-    if (isResponseValid && !hallucinated && hallucinated !== undefined) {
+    if (isResponseValid && !hallucinated) {
       // TODO cache summaries response after enabled
-      return summariesResponse?.answer
+      // return summariesResponse?.answer
+      try {
+        let answer
+        if (typeof summariesResponse === 'string') {
+          answer = JSON.parse(summariesResponse).answer
+        } else {
+          answer = summariesResponse.answer
+        }
+
+        if (typeof answer === 'string') {
+          answer = JSON.parse(answer)
+        }
+
+        if (answer && answer.items) {
+          items = answer.items
```

Comment:
At this point we already have the answer from the summary index, why is it not being returned as the answer?